bcda/api/requests_test.go

```diff
@@ -322,3 +425,47 @@ func (s *RequestsTestSuite) genASRequest() *http.Request {
 
 	return req.WithContext(ctx)
 }
+
+type MockService struct {
```

Comment:
We usually auto-generate mocks using mockery. But we can create a separate ticket to do this later.

Reply:
Actually, I think we should switch to using the mock service thats already in place to avoid stacking up tech debt. Look for usage of `mockSvc` in this file.